w5e1.py

```diff
@@ -0,0 +1,82 @@
+"w5e1" 
+import random
+import os
+
+# pokemon s1e1 -> Ash late to professor Oak so can't choose pokemon...
+from datetime import datetime, timedelta
+
+MONDAYS = 1
+
+
+def pokemon():
+    pokemons = ['Bulbasaur', 'Squirtle', 'Charmander']
+    starter_pokemon = random.choice(pokemons)
+    print(starter_pokemon)
+
+
+'''
+ This Is The Way
+ Write a function that gets a path to the folder,
+ and returns the list of all files that begin with the
+ "startsWith" letter sequence in that folder.
+'''
+
+
+def startsWith_files(folder, startsWith):
+    list_files = os.listdir(folder)
+    list_deep_files = []
+    for file in list_files:
+        if file.startswith(startsWith):
+            list_deep_files.append(file)
+    return list_deep_files
+
+
+# -----------------------------------------------------------------------#
+'''
+I have no vinaigrette
+Write software that receives as input from the user two dates in the configuration: YYYY-MM-DD.
+The software will grill a new date that is between the two dates that the user entered as input.
+For example, for inputs 1912-06-23 and 1954-06-07, a possible output is 1939-09-03.
+Since I only go to the grocery store on Mondays and I am a heavy consumer of vinaigrette sauce,
+ if the date falls on Monday, print: "I do not have vinaigrette!"
+'''
+
+
+def IHaveNoVinaigrette():
+    date1 = input("Enter first date: ")
+    while not is_valid_date(date1):
+        date1 = input("Enter first valid date: (YYYY-MM-DD.)")
+
+    date2 = input("Enter second date: ")
+    while not is_valid_date(date2):
+        date2 = input("Enter second valid date: (YYYY-MM-DD.)")
+
+    date1, date2 = datetime.strptime(min(date1, date2), '%Y-%m-%d'), datetime.strptime(max(date1, date2), '%Y-%m-%d')
```

Comment:
dont do that  - each variable in a separate line unless its a function that returns a tuple

Reply:
Even if I want to work on the same variables? And the goal was not to set new ones